lindeck-frontend/src/views/boody/deckPage/card/Card.jsx

```diff
@@ -0,0 +1,107 @@
+import "./Card.css"
+import React, {Component} from 'react';
+import CardSettingsBar from "./cardSettingsBar/CardSettingsBar";
+import TextEditDialog from "./textEditDialog/TextEditDialog";
+import ReactMarkdown from "react-markdown";
+import ReactCardFlip from "react-card-flip";
+import Paper from "@material-ui/core/Paper";
+import AnswerEditDialog from "./answerEditDialog/AnswerEditDialog";
+
+class Card extends Component {
```

Comment:
You should have components for all of 4 types of Cards

Reply:
Added comments